app/models/user.rb

```diff
@@ -5,4 +5,11 @@ class User < ActiveRecord::Base
   enum role: %w(default admin)
 
   has_many :orders
+
+  def set_order(session)
+    order = self.orders.create!(current_status: 'completed')
```

Comment:
Redundant `self` detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.